README/ReleaseNotes/v634/index.md

```diff
@@ -158,6 +158,8 @@ This should have minimal impact on users, since one should usual use Minuit 2 vi
 
 * Initial error values can be used for initializating the Hessian matrix to be used in Minuit2 minimization algorithms by setting the `RooMinimizer::Config` option `setInitialCovariance` to `true`. These values correspond to the diagonal entries of the initial covariance matrix.
 
+* `THStack:GetMinimum()` was not correct in case of negative contents.
```

Comment:
Why is this bullet under RooFit?

Reply:
Mistake. I meant to put in in the Histograms section